README.md

```diff
@@ -97,3 +97,8 @@ Therefore, if no endpoint is specified, a OneAgent is assumed to be running and
 
 Due to implementation details of the Go runtime and the OneAgent, it is currently not possible to read metadata on Unix/Linux systems,
 therefore OneAgent enrichment for Go only functions on Windows hosts at this time.
+
+## Non-string Attributes
```

Comment:
In the other language exporters we have a "somewhat" standard section for describing this: [Java](https://github.com/dynatrace-oss/opentelemetry-metric-java#typed-attributes-support), [Python](https://github.com/dynatrace-oss/opentelemetry-metric-python#typed-attributes-support). Maybe we should use the same to be consistent?